Add waitForAlertToBePresent method (#140)
wlsf82/protractor-helper: README.md

This method returns a boolean depending if the current url is different from the base url. No argument needed.
[Example](docs/EXAMPLES.md#iscurrenturldifferentfrombaseurl)

### `waitForAlertToBePresent`

> This method waits for an alert to be present in the DOM.

Reviewer: ```suggestion
This method waits for an alert to be present on the Page.
```

Author: done
